themes/docsy/layouts/partials/sidebar-tree.html

```diff
@@ -30,7 +30,7 @@
   </li>
   <ul>
     <li class="collapse {{ if $show }}show{{ end }}" id="{{ $sid }}">
-      {{ $pages := where (union $s.Pages $s.Sections).ByWeight ".Params.toc_hide" "!=" true }}
+      {{ $pages := where (union $s.Pages $s.Sections).ByWeight ".Params.page_hide" "!=" true }}
```

Comment:
Revert this change. We should not change the files in the `/themes` directory, as any changes here will be overridden next time we update our version of Docsy. This file is overridden by the one in the main `/layouts` directory.